### PR TITLE
rename create_sequential to sequence (#212) +small misc

### PR DESCRIPTION
**Pull request by gdementen**

### CODE REVIEW

File: larray/io/excel.py [hunk 1]
```diff
@@ -446,7 +446,7 @@ def __dir__(self):
 
         def __getattr__(self, key):
             if hasattr(LArray, key):
-                return getattr(self.__larray__(), key)
+                return getattr(LArray(self.__array__()), key)
             else:
```
Comment (alixdamman) [inline larray/io/excel.py]: Could you do the same in the method __str__ below?

**Comment by gdementen** [comment]
doh!

**Comment by alixdamman** [comment]
Did you run pytest this time? (to test xlwings)

**Comment by gdementen** [comment]
I run pytest everytime! But the flip side of the coin is that I rely too much on them. When they pass, I think it is ok. The Excel tests have a bad coverage so far...

**Comment by alixdamman** [comment]
OK to merge

**Pull request merged**